app-tracking-protection/vpn-internal/src/main/res/layout/activity_vpn_internal_settings.xml

```diff
@@ -71,49 +71,52 @@
-                    android:layout_height="wrap_content"
-                    android:text="Private DNS support"
-                    android:theme="@style/SettingsSwitchTheme"
+            <com.duckduckgo.mobile.android.ui.view.listitem.OneLineListItem
+                android:id="@+id/privateDnsToggle"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:primaryText="Private DNS support"
+                app:showSwitch="true"
             />
 
-            <androidx.appcompat.widget.SwitchCompat
-                    android:id="@+id/vpnUnderlyingNetworksToggle"
-                    style="@style/SettingsSwitch"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:text="Network Switching handling"
-                    android:theme="@style/SettingsSwitchTheme"
+            <com.duckduckgo.mobile.android.ui.view.listitem.OneLineListItem
+                android:id="@+id/vpnUnderlyingNetworksToggle"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:primaryText="Network Switching handling"
+                app:showSwitch="true"
             />
 
-            <androidx.appcompat.widget.SwitchCompat
-                    android:id="@+id/vpnAlwaysSetDNSToggle"
-                    style="@style/SettingsSwitch"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:text="Always set DNS"
-                    android:theme="@style/SettingsSwitchTheme"
+            <com.duckduckgo.mobile.android.ui.view.listitem.OneLineListItem
+                android:id="@+id/vpnAlwaysSetDNSToggle"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:primaryText="Always set DNS"
+                app:showSwitch="true"
             />
 
-            <androidx.appcompat.widget.SwitchCompat
-                    android:id="@+id/vpnConnectivityChecksToggle"
-                    style="@style/SettingsSwitch"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:text="VPN connectivity checks"
-                    android:theme="@style/SettingsSwitchTheme"
+            <com.duckduckgo.mobile.android.ui.view.listitem.OneLineListItem
+                android:id="@+id/vpnConnectivityChecksToggle"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:primaryText="VPN connectivity checks"
+                app:showSwitch="true"
+            />
+
+            <com.duckduckgo.mobile.android.ui.view.listitem.OneLineListItem
+                android:id="@+id/vpn_new_networking_layer_toggle"
```

Comment:
The `vpn_new_networking_layer_toggle` toggle is not needed anymore. It was removed in https://github.com/duckduckgo/Android/pull/2321/files#diff-f4d229785908cc736fd3edd2d6741744ae40a39a4402cded5753ea51ae4f1f3aL120